spec/features/admin/activity_spec.rb

```diff
@@ -134,10 +135,10 @@
 
   context "Comments" do
     scenario "Shows moderation activity on comments", :js do
-      debate = create(:debate)
+      debate = create(:debate, participatory_process: participatory_process)
       comment = create(:comment, commentable: debate)
 
-      visit debate_path(debate)
+      visit debate_path(debate, participatory_process_id: debate.participatory_process.slug)
```

Comment:
I would use `to_param` here, and it will delegate to `slug` or `id` accordingly. But shouldn't `debate_path` already do that by itself if you just pass it `debate.participatory_process`?